admin/spec/back-end/controllers/pupil-pin.spec.js

```diff
@@ -232,6 +232,7 @@ describe('pupilPin controller:', () => {
         const controller = require('../../../controllers/pupil-pin.js').postGeneratePins
 
         spyOn(checkStartService, 'prepareCheck')
+        spyOn(checkStartService, 'prepareCheck2')
```

Comment:
Tests seem to refer to and spy on the new version of the function, but don't actually cover any test cases.